### PR TITLE
Functional tests - Refacto Files, Discounts and Stocks tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Refacto Files, Discounts and Stocks tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25275)

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

File: tests/UI/campaigns/functional/BO/03_catalog/06_files/01_CRUDFile.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 require('module-alias/register');
 
+// Import expect from chai
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/BO/03_catalog/06_files/01_CRUDFile.js:3]: Never noticed that but, I feel like this comment is not useful at all, it's verbose enough to understand that you import expect from chai, wdyt @boubkerbribri ?

Comment (boubkerbribri) [reply tests/UI/campaigns/functional/BO/03_catalog/06_files/01_CRUDFile.js:3]: It's not useful, could be deleted later :+1:

**Comment by Progi1984** [comment]
## WAIT FOR 1780 RELEASE

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**